.github/workflows/managed.yml

```diff
@@ -76,9 +76,6 @@ jobs:
         continue-on-error: true
         run: docker exec -t pmm-server supervisorctl status
 
-      - name: Refresh yum cache for tests
```

Comment:
This is no longer relevant, it only slows things down. We will save from 60 to 80 seconds of build time.